src/app/modules/shared-components/components/error/error.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="wrapper">
```

Comment:
I found other components that should probably use the `<alg-error />` component, can you check if it is indeed the case or not :pray: ?

- [ ] item-children-edit.component.html #l113 ([test link](https://dev.algorea.org/branch/feature/error-component/en/#/activities/by-id/1980584647557587953;path=4702,4102;attempId=0/edit))
- [ ] item-remove-button.component.html #l17 ([test link](https://dev.algorea.org/branch/feature/error-component/en/#/activities/by-id/1980584647557587953;path=4702,4102;attempId=0/edit/advanced-parameters))
- [ ] item-by-id.component.html #l10 ([test link](https://dev.algorea.org/branch/feature/error-component/en/#/activities/by-id/39530140456452546;path=4702,4102,1980584647557587953;attempId=0/edit/advanced-parameters))
- [ ] pending-requests.component.html #l12 ([test link](https://dev.algorea.org/branch/feature/error-component/en/#/groups/mine), expand "pending request" or "pending invitations")

Reply:
item-children-edit.component.html - Fixed
pending-requests.component.html - Fixed
item-remove-button.component.html - Hmm, I've add <alg-error there
item-by-id.component.html - I guess there is too specific template